.github/workflows/maven-deploy.yml

```diff
@@ -95,7 +103,7 @@ jobs:
         env:
           PRINT_PAYLOAD: true
           CLUSTER: ${{ inputs.cluster }}
-          IMAGE: ${{ steps.login.outputs.registry }}/${{ github.repository }}${{ inputs.image_suffix }}:${{ inputs.image }}
+          IMAGE: ${{ inputs.override_image_name != '' && inputs.override_image_name || format('{0}/{1}{2}:{3}', steps.login.outputs.registry, github.repository, inputs.image_suffix, inputs.image) }}
```

Comment:
Trengs begge sjekkene inputs.override_image_name != '' && inputs.override_image_name når det alltid settes en defaultverdi ?

Reply:
tror kanskje tom string ikke regnes som false, for vi gjør det samme på linje 83